assets/src/stories-editor/helpers/index.js

```diff
@@ -2143,3 +2143,29 @@ export const isPageBlock = ( clientId ) => {
 	const block = getBlock( clientId );
 	return block && 'amp/amp-story-page' === block.name;
 };
+
+/**
+ * Parses drag & drop events to ensure the event contains valid transfer data.
+ *
+ * @param {Object} event
+ * @return {Object} Parsed event data.
+ */
+export const parseDropEvent = ( event ) => {
```

Comment:
There's also a `parseDropEvent` helper in the `ReordererItem` component with a slightly different `result` object. Can they be merged?